src/main/java/io/github/darealturtywurty/ancientology/common/blocks/TestBlock.java

```diff
@@ -0,0 +1,11 @@
+package io.github.darealturtywurty.ancientology.common.blocks;
+
+import net.minecraft.world.level.block.Block;
+
+public class TestBlock extends Block {
```

Comment:
I Would say remove the test block and leave the commented out example in the Block Init Class